esp/services/ws_esdlconfig/ws_esdlconfigservice.hpp

```diff
@@ -66,8 +65,12 @@ class CWsESDLConfigEx : public CWsESDLConfig
 
     bool attachServiceToDali() override
     {
-        m_isDetachedFromDali = false;
-        return true;
+        m_esdlStore.setown(createEsdlCentralStore());
```

Comment:
The store doesn't need to be recreated every time this is called, does it? If it makes sense to retry store creation after its initial failure in init, only do it if there is no current store.

Reply:
I've made the assignment conditional. @afishbeck do you have any comments about this issue or the PR overall?